src/wasm.h

```diff
@@ -1734,10 +1734,10 @@ class Module {
   Global* addGlobal(Global* curr);
   Event* addEvent(Event* curr);
 
-  Export* addExport(std::unique_ptr<Export> curr);
-  Function* addFunction(std::unique_ptr<Function> curr);
-  Global* addGlobal(std::unique_ptr<Global> curr);
-  Event* addEvent(std::unique_ptr<Event> curr);
+  Export* addExport(std::unique_ptr<Export>&& curr);
+  Function* addFunction(std::unique_ptr<Function>&& curr);
+  Global* addGlobal(std::unique_ptr<Global>&& curr);
+  Event* addEvent(std::unique_ptr<Event>&& curr);
```

Comment:
Why this?

Reply:
I wanted to use the overloads of these functions that take `std::unique_ptr` rather than raw pointers, since allocating an object with `std::make_unique` up front is more efficient than wrapping a raw pointer into a `std::unique_ptr`. However, `std::unique_ptr` can't be copied, so it wasn't actually possible to call these overloads the way I needed to without changing them to take rvalue references.